### PR TITLE
Archive listing employes

### PR DESCRIPTION
**Pull request by michalsz**

### CODE REVIEW

File: app/javascript/views/employees/Archived.vue [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<template>
+  <section>
+    <div class="panel__content">
+      <v-container grid-list-lg fluid>
+        <archived-table
+          :employees="employees"
+          :loading="loading"
+          :editable="user.isAdmin"
```
Comment (schodevio) [inline app/javascript/views/employees/Archived.vue]: Archived Employees won't be editable so this binding can be skipped

File: spec/controllers/v2/employees_controller_spec.rb [hunk 2]
```diff
@@ -520,4 +520,20 @@
       end
     end
   end
+
+  describe '#archived' do
```
Comment (schodevio) [inline spec/controllers/v2/employees_controller_spec.rb]: Please add specs for unauthorized and unauthenticated cases

File: app/javascript/components/employees/ArchivedTable.vue [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      type: Boolean,
+      required: true,
+      default: false
+    }
+  },
+  data() {
+    return {
+      search: '',
+      headers: [
+        {
```
Comment (schodevio) [inline app/javascript/components/employees/ArchivedTable.vue]: this table column can be removed as we don't have edit action

File: app/javascript/router/router.js [hunk 4]
```diff
@@ -46,6 +46,12 @@ export default new Router({
               component: () => import(/* webpackChunkName: 'employees_overview' */ '@views/employees/Overview'),
               beforeEnter: authorizationGuard
             },
+            {
+              path: 'archived',
+              name: 'employees_archived_path',
+              component: () => import(/* webpackChunkName: 'employees_overview' */ '@views/employees/Archived'),
```
[the remark this answers is not in the document]
Comment (michalsz) [reply app/javascript/router/router.js]: Updated

File: spec/controllers/v2/employees_controller_spec.rb (hunk 2 above)
Comment (michalsz) [reply spec/controllers/v2/employees_controller_spec.rb]: I added

**Pull request merged**